pom.xml

```diff
@@ -627,7 +627,7 @@
       <dependency>
         <groupId>org.lz4</groupId>
         <artifactId>lz4-java</artifactId>
-        <version>1.6.0</version>
+        <version>1.7.0</version>
```

Comment:
Could you update the dependency files with `dev/test-dependencies.sh --replace-manifest`?